server.js

```diff
@@ -1,30 +1,58 @@
-import express from 'express'
+import express, { response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import booksData from './data/books.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+// Defines the port the app will run on (8080 is default)
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// Endpoints/route that return a collection of results
+app.get('/', (request, response) => {
+  response.send('Welcome to my book API')
+})
+
+// Endpoint/route that return the first 200 books in the data 
+app.get('/books', (request, response) => {
+  const firstBooks = booksData.slice(0, 200)
+  
+    response.json(firstBooks)
+})
+
+// Endpoint/route that return a single book based on the ID
+app.get('/books/book/:bookID', (request, response) => {
+  const bookID = request.params.bookID
+  const specificBook = booksData.find((specificBook) => specificBook.bookID === +bookID)
+    
+    response.json(specificBook)
+})
+
+// Endpoint/route that return a list of books by a specific author
+app.get('/books/search', (request, response) => {
+  const author = request.query.author
+  const filteredAuthors = booksData.filter((item) => item.authors.toLowerCase().includes(author.toLowerCase()))
+    
+    response.json(filteredAuthors)
+})
```

Comment:
Nice with some filtering! Would be even nicer if this was a query under your main endpoint though, as endpoints should be named after what they return and not after filtering options etc. So something like:
`/books?author=rowling`